### DISCUSSION TITLE
Deploys

### DISCUSSION THREAD

**Discussion by pedroleone** [pedroleone/osr-srd#11] [General]
O site atualmente está hospedado no Netlify. O netlify tem uma ferramenta bastante interessante de auto-deploy com os commits do Github, entretanto tem um limite relativamente baixo de tempo de processamento. Ao exceder este tempo há uma cobrança automática, e eu tive problemas no passado com o excesso de commits estourando este limite rapidamente. Por esse motivo o auto deploy foi desativado e por enquanto só eu consigo fazer o deploy.

Tentei adicionar o @nerun como membro do time no Netlify para ele conseguir fazer o deploy, mas isso custaria 15 dólares por mês, portanto está fora de cogitação. 

Desta forma, estou me progamando para fazer um deploy por semana, normalmente no início da semana. Caso seja necessário antes disso, me avisem que eu faço um deploy fora da programação.

**Comment by nerun** [comment]
> Desta forma, estou me programando para fazer um deploy por semana, normalmente no início da semana. Caso seja necessário antes disso, me avisem que eu faço um deploy fora da programação.

Boa! E semanalmente é uma ótima periodicidade, dá tempo do pessoal ver e corrigir qualquer coisa antes de ir ao ar. Assim é melhor!

**Comment by nerun** [comment]
Ainda bem que você decidiu assim. Tenho editado alguns textos diretamente pelo GitHub, porque estou no celular! O resultado é um monte de *commit*, porque ele só permite uma edição de cada vez.
